bundles/org.openhab.binding.goecharger/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -38,6 +40,10 @@
 				<description>The IP address of the Go-eCharger</description>
 				<context>network-address</context>
 			</parameter>
+			<parameter name="apiVersion" type="integer" required="true">
```

Comment:
Consider if this parameter could be optional so you could default to 1. This would provide backwards compatibility for existing users.